### DISCUSSION TITLE
What does this CheckSumTag error mean?

### DISCUSSION THREAD

**Discussion by helloissacli** [TimelordUK/jspurefix#89] [Q&A] [answered]
Hi @TimelordUK, thank you for your excellent work on the jspurefix package. 

I encountered the following error while using it: `CheckSumTag: [121] expected after 140.`

Could you kindly provide some insight into what this error might indicate and how to resolve it?

I appreciate your time and assistance.

**Comment by TimelordUK** [comment, marked as answer]
can you check the message in quesion and see where the 10= tag is placed, this sounds like the parser has found the 10= tag in a position it does not expect and has thrown an error as a result.  If you remove sensitive information you can copy message here to take a look. its saying at position 121 it sees 10= yet it should not be found until position 140

**Comment by TimelordUK** [comment]
8=FIX.5.0SP2|9=0000178|35=AE|49=*|56=init-comp|34=473|57=fix|52=20241010-12:34:24.832|571=100469|487=0|856=0|828=0|17=600469|570=N|55=Steel|48=Steel|32=1000|31=100|75=20241010|60=20241010-12:34:24.828|10=049|

where is this tag |10=xxx|
